Formula/a/autopsy.rb

```diff
@@ -1,9 +1,11 @@
 class Autopsy < Formula
   desc "Graphical interface to Sleuth Kit investigation tools"
-  homepage "https://www.sleuthkit.org/autopsy/index.php"
-  url "https://downloads.sourceforge.net/project/autopsy/autopsy/2.24/autopsy-2.24.tar.gz"
-  sha256 "ab787f519942783d43a561d12be0554587f11f22bc55ab79d34d8da703edc09e"
+  homepage "https://www.autopsy.com/"
+  url "https://github.com/sleuthkit/autopsy.git",
+      tag:      "autopsy-4.21.0",
+      revision: "c23b4fd71d8966a80a8d6d2b5d1a979522dfcc49"
```

Comment:
Do we need a git clone? Can we use a checkout instead?

Reply:
@carlocab I could use the zip ig that would be better than a clone, should I do that?